src/map/script.cpp

```diff
@@ -9797,6 +9797,29 @@ BUILDIN_FUNC(autobonus3)
 	return SCRIPT_CMD_SUCCESS;
 }
 
+BUILDIN_FUNC(skill_plagiarism)
+{
+	TBL_PC* sd;
+	uint16 skill_id = script_getnum(st, 2);
+	uint16 skill_lv = script_getnum(st, 3);
+
+	if (script_rid2sd(sd))
+		script_pushint(st, pc_skill_plagiarism(sd, skill_id, skill_lv));
```

Comment:
```suggestion
		script_pushint(st, pc_skill_plagiarism(sd, script_getnum(st, 2), script_getnum(st, 3)));
```
Since no checks are made against skill_id or skill_lv here, we can save time/memory and just directly send it in.